### ISSUE TITLE
Framework is automatically adding arguments with value 'undefined'

### ISSUE THREAD

**Issue by maxp-hover** [EasyGraphQL/easygraphql-load-tester#50]
I have a query, where all the arguments are optional. The `easygraphql-load-tester` library isn't complaining about this, but it is automatically adding all of those arguments with value undefined. 

For example if I have this schema:

```
query GetJobs(
    $updatedAfter: ISO8601DateTime
    $search: String
    $createdBefore: ISO8601DateTime
) {
  // Response details omitted here
}
```

It should work even if I omit all those args, none of them are mandatory. This is an extract of my `config.json` file:

```
"args": {
  "jobs": {},
}
```

Yet when I actually look at the query was sent, it's including all those arguments as 'undefined':

```
{
  jobs(
    createdBefore: "undefined",
    search: "undefined",
    updatedAfter: "undefined"
  ) {
    // response details omitted here
  }
}
```

This results in errors:

> "Argument 'createdBefore' on Field 'jobs' has an invalid value. Expected type 'ISO8601DateTime'.",
> "Argument 'updatedAfter' on Field 'jobs' has an invalid value. Expected type 'ISO8601DateTime'."

As a workaround, I have patched the chunk of code that builds the `queryArgs`:

```
    if (
      userArg != undefined &&
      userArg != 'undefined'
    ) {
      queryArgs.push(`${arg.name}: ${userArg}`)
    }
```

I don't want to push the arg if its value is set to `undefined` or `"undefined"`.

What do you think about this? Should we try and bring this change into the library via a PR? Is there a better way to solve the problem, or has nobody else found it to be an issue?

**Comment by estrada9166** [comment]
@maxp-hover Thanks for creating this issue.

Sorry the delay. This may be fixed in the [`v1.0.0-beta.2`](https://www.npmjs.com/package/easygraphql-load-tester/v/1.0.0-beta.2)
If you can give it a try it'll be awesome!

**Comment by estrada9166** [comment]
This should be fixed in [`v2.0.1`](https://www.npmjs.com/package/easygraphql-load-tester/v/2.0.1)
